README.md

```diff
@@ -57,6 +57,29 @@ pub fn main() {
 gleam add gleam_pgo
 ```
 
+## Support of connection URI
+
+Configuring a Postgres connection is done by using `Config` type in `gleam/pgo`.
+To facilitate connection, and to provide easy integration with the rest of the
+Postgres ecosystem, `gleam_pgo` provides handling of
+[connection URI as defined by Postgres](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS). Call `gleam/pgo.url_config` with your connection URI,
+and in case it's correct against the Postgres standard, your `Config` will be
+automatically generated!
```

Comment:
Could you include an example of how this is done? Perhaps reading environment variables with envoy? It can be hard to tell what code to write from prose.